src/import/sst_service.rs

```diff
@@ -1240,6 +1307,27 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
         RawChunk,
         new_raw_writer
     );
+
+    fn suspend_import_rpc(
+        &mut self,
+        ctx: RpcContext<'_>,
+        req: SuspendImportRpcRequest,
+        sink: UnarySink<SuspendImportRpcResponse>,
+    ) {
+        let label = "suspend_import_rpc";
+        let timer = Instant::now_coarse();
+
+        let suspended = if req.should_suspend_imports {
+            info!("suspend incoming import RPCs."; "for_second" => req.get_duration_in_secs(), "caller" => req.get_caller());
+            self.suspend_requests(Duration::from_secs(req.get_duration_in_secs()))
```

Comment:
I think we need a limitation check for `req.get_duration_in_secs()`. for example 30mins.

Reply:
It is reasonable for suspend for a long time. (for example, longer ttl usually allow us to disconnect to TiKV for a longer time) I think if we need to check, the max should be at least about 5 ~ 6 hours.